webpack/components/surveys/SurveyShow.js

```diff
@@ -429,6 +436,12 @@ class SurveyShow extends Component {
                 </div>
                 }
                 { this.props.survey.parent &&
+                <div className="box-content">
+                  <strong>Published By: </strong>
+                  {this.props.survey.publishedBy.email}
+                </div>
```

Comment:
Not surew why lines 438-443 are here, they duplicate 413-418 but the check at the top is wrong. if this.props.survey.publishedBy is undefined this will crash the page because line 438 is incorrect (should look like line 413)

Reply:
Removed, tests are passing.